### ISSUE TITLE
Generate command after key rotation in GETTING STARTED guide doesn't quite work.

### ISSUE THREAD

**Issue by daidoji** [hyperledger-labs/did-webs-resolver#69]
In this section: https://github.com/hyperledger-labs/did-webs-resolver/blob/main/GETTING_STARTED.md#example-key-rotation

The key rotation worked fine but that generate command doesn't work (and also uses a different domain for some reason).  This was also performed at the end of the long sequence of "GETTING_STARTED.md" guide.

```
webs:/usr/local/var/webs/volume/dkr/examples# dkr did webs generate --name controller --did did:webs:blockchainbird.org:did-webs:EG8GsKYdICKs-zI6odM6tvCmxRT2J-7UkZFqA77agtb8 --oobi http://witnesshost:5642/oobi/EG8GsKYdICKs-zI6odM6tvCmxRT2J-7UkZFqA77agtb8/witness/BBilc4-L3tFUnfM_wJr4S4OJanAv_VmF_dJNN6vkf2Ha
usage: dkr [-h] command ...
dkr: error: unrecognized arguments: --oobi http://witnesshost:5642/oobi/EG8GsKYdICKs-zI6odM6tvCmxRT2J-7UkZFqA77agtb8/witness/BBilc4-L3tFUnfM_wJr4S4OJanAv_VmF_dJNN6vkf2Ha
```

**Comment by kentbull** [comment]
Will you try generation again with the latest code from the https://github.com/GLEIF-IT/did-webs-resolver fork? The Getting Started guide now also references a Bash script that shows the end to end process that may speed things up for you.

If the generation command is still using a different domain then that is a bug.

The documentation and design of the getting started guide has significantly changed and you will no longer use the directory `/usr/local/var/webs/volume/dkr/examples`. Instead, for the Docker Compose setup you can use the `dws-shell` container to perform `dws did webs generate ...`, or use the script in `did-webs-resolver/local` for a local setup.

**Comment by daidoji** [comment]
@kentbull actually do you want me to debug this getting_started.md guide?  Its been a long time but I think this error came up when I was attempting to use the manual walkthrough documentation.  I see that its still "work in progress" https://github.com/GLEIF-IT/did-webs-resolver/blob/main/docs/getting_started.md#local-command-line-flow-best-for-development-and-debugging 

If this isn't shored up maybe better to fix this issue after it is.

**Comment by kentbull** [comment]
That's right, it's not quite ready for review. I did a bit of work to upgrade it yet haven't reviewed the whole thing end to end in a while. I'll leave it be for now as I'm spread pretty thin at the moment.
